src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -56,22 +68,11 @@ def update_data(self):
                 break
 
             ent = self.db_conn.get_all(mibs.ASIC_DB, s, blocking=True)
-            port_oid = ent[b"SAI_FDB_ENTRY_ATTR_PORT_ID"]
-            if port_oid.startswith(b"oid:0x"):
-                port_oid = port_oid[6:]
-
-            ## Note: broadcom SAI 0.94 behavior
-            ## Some FDB_ENTRY's port_oid starts with 0x2 is LAG. The whole FDB_ENTRY is not expected,
-            ## and should be ignored here
-            ## TODO: revisit logic here for LAG in VLAN
-            if port_oid not in self.if_id_map:
-                ## Reduce the logger amount by remember it in a set
-                if port_oid not in self.invalid_port_oids:
-                    self.invalid_port_oids.add(port_oid)
-                    mibs.logger.info("SyncD 'ASIC_DB' includes a FDB_ENTRY '{}' with an invalid interface id '{}'.".format(fdb_str, port_oid))
-                continue
+            bridge_port_id = ent[b"SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"][6:]
+            port_id = self.if_bpid_map[bridge_port_id]
```

Comment:
Does it also cover the LAG case?

Reply:
good catch. I think right now we don't support LAG. I'll add todo here. We don't have such testbed setup in our env. I'll sync with Alibaba to check if they could help enhance this scenario.